test/node/TraceSpec.purs

```diff
@@ -0,0 +1,42 @@
+module TraceSpec where
+
+import Prelude
+
+import Control.Monad.Cleanup (execCleanupT, runCleanupT)
+import Data.Tuple (Tuple(..))
+import Debug.Specular (traceDyn, traceEvent)
+import Specular.FRP (holdDyn, newEvent, subscribeDyn_, subscribeEvent_)
+import Test.Spec (Spec, describe, it)
+import Test.Spec.Runner (RunnerEffects)
+import Test.Utils (ioSync, withLeakCheck)
+
+spec :: forall eff. Spec (RunnerEffects eff) Unit
+spec = describe "Trace" $ do
```

Comment:
Please rewrite the tests so that they actually test the functionality, i.e. use `traceEventIO` and `traceDynIO`.
Make sure to test for the bad double-tracing behavior you observed.